app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -342,7 +429,50 @@ export default function RegistrationActions({
   );
 }
 
-function MoveAction({
+function SummaryTable({
+  partitionedSelectedIds,
+  partitionedRegistrations,
+  partitionedMaximums,
+  selectedCount,
+  registrationCount,
+  withSelectedCounts,
+  withMaximums,
+}) {
+  return (
+    <Table basic="very">
```

Comment:
All of the below table components can use dot notation on the original `Table` import, for example `Table.Header` or `Table.HeaderCell` (and others). This saves you a lot of import statements at the top.